site/content/docs/4.3/examples/offcanvas/index.html

```diff
@@ -38,9 +38,13 @@
           </div>
         </li>
       </ul>
-      <form class="form-inline my-2 my-lg-0">
-        <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+      <form class="row gx-2">
```

Comment:
Maybe related to this exact line: the navbar container is broken (content touches the edge of the viewport). Happens on the navbar-bottom example, too.